Packs/Slack/Integrations/Slack_IAM/README.md

```diff
@@ -0,0 +1,475 @@
+Note: This integration should be used as part of our IAM premium pack. For further details, visit our IAM pack documentation.
+
+Integrate with Slack's services to execute CRUD operations for employee lifecycle processes.
+For more information, please refer to the [Identity Lifecycle Management article](https://xsoar.pan.dev/docs/reference/articles/identity-lifecycle-management).
+
+## Configure Slack IAM on Cortex XSOAR
+
+1. Navigate to **Settings** > **Integrations** > **Servers & Services**.
+2. Search for Slack IAM.
+3. Click **Add instance** to create and configure a new integration instance.
+
+    | **Parameter** | **Description** | **Required** |
+    | --- | --- | --- |
+    | access_token | Access Token | True |
+    | insecure | Trust any certificate \(not secure\) | False |
+    | proxy | Use system proxy settings | False |
+    | create_user_enabled | iam-create-user Command Enabled | False |
```

Comment:
all the default values for these params are `True`

Reply:
Which Params exactly? 
I checked the rest of the integrations, it is `False` there as well.